schemas/orders.js

```diff
@@ -0,0 +1,19 @@
+const mongoose = require("mongoose");
```

Comment:
Probably it's better to call folder as models. Since export of this files is mongoose model